app/views/homepage/_grid_item.haml

```diff
@@ -1,11 +1,12 @@
-.home-fluid-thumbnail-grid-item
-  %div
-    -# Listing image
-    = render :partial => "layouts/grid_item_listing_image", :locals => {:listing => listing, :modifier_class => ""}
+- cache(listing) do
```

Comment:
This way the cache key is generated by combining the model name, id and `updated_at` timestamp, as described here: [Fragment caching](http://guides.rubyonrails.org/caching_with_rails.html#fragment-caching)

> Behind the scenes, a method called cache_key will be invoked on the model and it returns a string like products/23-20130109142513. The cache key includes the model name, the id and finally the updated_at timestamp. Thus it will automatically generate a new fragment when the product is updated because the key changes.